interfaces/src/main/java/dotty/tools/dotc/interfaces/CompilerCallback.java

```diff
@@ -0,0 +1,30 @@
+package dotty.tools.dotc.interfaces;
+
+import java.io.File;
+
+/** Set of callbacks called in response to events during the compilation process.
+ *
+ *  You should implement this interface if you want to react to one or more of
```

Comment:
JavaDoc style is different than Scaladoc. See http://www.oracle.com/technetwork/articles/java/index-137868.html#format
It should therefore be:
```java
/**
 * Set of callbacks ...
 *
 * You should ...
 * ...
 */
```
(same everywhere there are JavaDocs, obviously.)